### ISSUE TITLE
About trajectory analysis

### ISSUE THREAD

**Issue by evenDDDDD** [hongleir/SpaceFlow#10]
Congratulations for this nice work!
I wonder if it is possible to use SpaceFlow for trajectory analysis of spatial data from merfish (single-cell level)？

Thanks!

**Comment by hongleir** [comment]
Yes, it should be able to do that, as long as the data contains both spatial coordinates and expression, but the result may not ideal since the number of genes might be less than usual.
